examples/basic_mysql_connection/index.js

```diff
@@ -8,13 +8,22 @@ var connection = mysql.createConnection({
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
-
-connection.query('SELECT * FROM todos', function (error, results, fields) {
+let query=['select count(*) as items from todos',
+           'select * from todos where done',
+           'select * from todos where !done',
+					 'select * from todos order by due DESC',
```

Comment:
Forgot to mention - this doesn't actually accomplish what the question asked for (though it wasn't super clear). I was looking for all todo items sorted with the most recently added ones first.